examples/sn-react-component-docs/.storybook/manager.js

```diff
@@ -1,7 +1,14 @@
+import { addons } from '@storybook/addons'
 import '@storybook/addon-notes/register'
 import '@storybook/addon-knobs/register'
 import '@storybook/addon-actions/register'
 import '@storybook/addon-links/register'
 import '@storybook/addon-options/register'
 import '@storybook/addon-a11y/register'
 import '@storybook/addon-viewport/register'
+
+addons.setConfig({
+  sidebar: {
+    showRoots: false,
+  },
+})
```

Comment:
is it needed?

Reply:
yeah, otherwise it will display FieldControls as a root (I tried it and it was not pretty), now we want folder icon ... 
here is the example: https://storybook.js.org/docs/riot/configure/sidebar-and-urls#roots